templates/bot/ts/command-and-response/src/helloworldCommandHandler.ts

```diff
@@ -17,13 +16,36 @@ export class HelloWorldCommandHandler implements TeamsFxBotCommandHandler {
   ): Promise<string | Partial<Activity> | void> {
     console.log(`Bot received message: ${message.text}`);
 
-    // Render your adaptive card for reply message
-    const cardData: CardData = {
-      title: "Your Hello World Bot is Running",
-      body: "Congratulations! Your hello world bot is running. Click the documentation below to learn more about Bots and the Teams Toolkit.",
+    // An example Adaptive Card that defines the response message of this helloWorld command.
+    const cardJson = {
```

Comment:
I would consider keeping this as an external file rather than bringing it inline.

You lose the context that there is a card definition from briefly looking at the project structure and also lose the ability to view the card in the Adaptive Cards Designer VSCode extension.